operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -74,6 +74,18 @@ Create the name of the service account
 {{- end -}}
 {{- end -}}
 
+{{/*
+Create the name of the ruler service account
+*/}}
+{{- define "mimir.ruler.serviceAccountName" -}}
+{{- if .Values.ruler.serviceAccount.create -}}
+{{- $sa := default (include "mimir.fullname" .) .Values.serviceAccount.name }}
```

Comment:
should be this be `.Values.ruler.serviceAccount.name` instead of `.Values.serviceAccount.name`?